test/functional/editor/jump_spec.lua

```diff
@@ -182,6 +182,109 @@ describe("jumpoptions=stack behaves like 'tagstack'", function()
   end)
 end)
 
+describe('buffer deletion', function()
+  local base_file = 'Xtest-functional-buffer-deletion'
+  local file1 = base_file..'1'
+  local file2 = base_file..'2'
+  local file3 = base_file..'3'
+  local base_content = 'text'
+  local content1 = base_content..'1'
+  local content2 = base_content..'2'
+  local content3 = base_content..'3'
+  local function delete_buf(file)
+    local bufnr = funcs.bufnr(file)
+    command('bw '..bufnr)
```

Comment:
use full command names in code. bwipout also works with buffer names, is bufnr() needed?

```suggestion
    command('bwipeout '..file)
```

Reply:
I didn't realize, thank you for the info!

Reply:
I didn't realize, thanks for pointing it out!